backend-lib/src/main/rust/utils/exception.rs

```diff
@@ -52,7 +52,7 @@ pub fn unwrap_exc_or<T>(env: &JNIEnv, res: ExceptionResult<T>, error_val: T) ->
 
```

Comment:
Delete the commented-out code or change it to use `env: &mut JNIEnv` so that it would compile. (Probably delete it: writing `unwrap_exc_or_default` is not much more concise than just passing the default, and I prefer the explicitness of the latter.)